kitchen-tests/cookbooks/end_to_end/recipes/_macos_userdefaults.rb

```diff
@@ -11,16 +11,22 @@
   value 0
 end
 
-macos_userdefaults "enable macOS firewall" do
+macos_userdefaults "Enable macOS firewall" do
   domain "/Library/Preferences/com.apple.alf"
   key "globalstate"
   value "1"
   type "int"
 end
 
-macos_userdefaults "set dock size" do
+macos_userdefaults "Set the dock size" do
   domain "com.apple.dock"
   type "integer"
   key "tilesize"
   value "20"
 end
+
+macos_userdefaults "value with space" do
```

Comment:
lamont says this is blocking unless we capitalize this

Reply:
literally unplayable

Reply:
Fixed